<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizScreen3</class>
 <widget class="QWidget" name="QuizScreen3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>630</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="quizLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Module 3 Quiz: What Can You Do to Decrease Your Risk?</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>1141</width>
     <height>541</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1125</width>
      <height>780</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>780</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>861</width>
       <height>754</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="q1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What is a recommended precaution to decrease the risk of avalanches?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Engage in winter sports immediately after snowfall</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Avoid outdoor activities during and shortly after heavy snowfall</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Avoid slopes with a lot of trees</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d1">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hike on steep slopes to enjoy the snow</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question1</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q2">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What slope angle should individuals avoid to reduce the risk of avalanches?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Slopes with pitches greater than 15 degrees</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Slopes with pitches greater than 25 degrees</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Only flat terrains are safe</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Any slope angle is acceptable with proper equipment</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question2</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q3">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>What is the purpose of digging snow pits in avalanche risk management?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>To create a shelter during snowstorms</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>To identify potential avalanche problems</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>To build snow sculptures</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>To mark the boundaries of safe areas</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question3</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="q4">
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>How does assessing the season's history contribute to snowpack stability evaluation?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="a4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It helps determine the best time for winter sports</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It allows for a more accurate assessment of potential avalanche problems</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It provides information on the temperature of the snowpack</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="d4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>It is irrelevant to avalanche risk assessment</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">question4</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>630</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>630</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="submitMessage">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>20</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="question1"/>
  <buttongroup name="question3"/>
  <buttongroup name="question4"/>
  <buttongroup name="question2"/>
 </buttongroups>
</ui>
